features/support/env.js

```diff
@@ -44,6 +44,8 @@ module.exports = function () {
 
         this.OSRM_PORT = process.env.OSRM_PORT && parseInt(process.env.OSRM_PORT) || 5000;
         this.HOST = 'http://127.0.0.1:' + this.OSRM_PORT;
+        
+        this.OSRM_PROFILE = process.env.PROFILE
```

Comment:
should be here `process.env.OSRM_PROFILE`?

Reply:
yes, sorry for the copy paste error